ArcGISRuntimeSDKQt_CppSamples/Maps/ReadGeopackage/README.md

```diff
@@ -0,0 +1,30 @@
+# Read geopackage
+
+This sample demonstrates how to read rasters and feature tables from geoPackages to show them as layers in a map.
+
+![](screenshot.png)
+
+## How to use the sample
+Use the top-left menu to select individual geopackage layers to add to the map. Unselect them to hide them.
+
+## How it works
+1. Create an empty menu UI using QML, along with empty `listView` and `listModel` objects.
+2. Create a `GeoPackage` QML object within the `listView` and load it locally, or download if remote.
+3. Iterate through the geopackage, once for the `featureTables` layers and once for the `raster` layers. In each iteration, create an `ArcGISRuntime` Object that constructs the appropriate layer.
+4. Still within the iteration, create a JSON object that has a `lyr` (layer) property and a `name` (name) key, both set to the corresponding `ArcGISRuntime` object values. Append these to the `listModel`.
```

Comment:
Change `create a JSON object that has a lyr (layer) property and a name (name) key, both set to the corresponding ArcGISRuntime object values` to `create a QVariantMap that has a lyr (layer object) and name (layer name) keys` and set the corresponding values for these keys.

Reply:
> Append these to the listModel

This seems more relevant to the QML sample.  In C++ you are appending the QVariantMaps to a QVariantList to be used as the data model.